### ISSUE TITLE
Banjo Kazooie Intro Music Out Of Sync

### ISSUE THREAD

**Issue by Toonerz** [project64/project64#1306]
This has been an issue that has been in project 64 for many years, and I feel that this issue needs to be addressed and fixed after all of this time. It's not just the intro music either, most of the  cut scene's that take place seem to play slower and the audio does not sync up like it would on the real console. Is there any way to resolve this long time issue?

**Comment by Jj0YzL5nvJ** [comment]
Check "Sync game to audio", that worked in 1.6.

**Comment by AmbientMalice** [comment]
Sync game to audio is a legacy setting that conflicts with the current emulator configuration.

**Comment by Toonerz** [comment]
I would love to say this fixed my issue but the timing is still off and not like the real console. So far the only known emulator to have the correct timing (or close to it) is dk64 on wii u virtual console injected with the banjo kazooie rom.

**Comment by Jj0YzL5nvJ** [comment]
I don't have complaints regarding Banjo Kazooie with mupen64plus + GLideN64 on Xubunutu 16.04.2.
[Try this](http://ngemu.com/threads/what-is-the-best-most-accurate-plugins-for-project-64-1-6.183961/#post-2504345), the last time I used it worked fine... except the "puzzle effect".

**Comment by Toonerz** [comment]
Maybe i should be more specific, take a look at the intro where they introduce themselves and the opening cutscene on a real console: https://www.youtube.com/watch?v=gYRL25GXTQE&index=1&list=PL45FXiU5g2Q-Qg4e1xr2p7V-jwhutk5TA As you might see, that is how the intro is supposed to play out, but on every emulator i have tried, it has never been close to the real hard ware, well, aside from the aformentioned virtual console inject on wii u.

**Comment by Jj0YzL5nvJ** [comment]
Yeah, I understand perfectly. I played it myself in my console (N64).

 `Project64 1.6 + Glide64 Final + Jabo's audio 1.6.1 + uoRDB v4.23` with "Sync game to audio" can play the intro perfectly, except for the effect on the jigsaw puzzle's.
 `mupen64plus + GLideN64 + audio-sdl + rsp-hle` in "Linux" with Mesa drivers can emulate the whole game near to perfection, that includes the intro.

 The only problems to naked eye are those related to text (in many RARE games). A pink color behind some spirits in the pause menu (mupen64plus). And the "black borders" ([overscan](https://www.reddit.com/r/n64/comments/4w1q62/conkers_bad_fur_day_resolution/)), which can be considered as accurate emulation (GLideN64).

It's all, I do not know what else to tell you. Both options work on my toasters.

**Comment by Toonerz** [comment]
Cartainly much closer, but is still slightly off sync, and the part where they introduce them selves with the instruments seems to start a bit slower than how it would on the n64. I know that the issue is rather minor, but it is an issue the bothers me, due to having grown up with this game on the original console, and knowing how everything is supposed to play out.

**Comment by oddMLan** [comment]
Zilmar is doing console tests again, which is great, but he's mostly busy refactoring the graphics plugin. I would also like to see the core timing improved. Accurate without being CEN64, of course, that would be slow.

**Comment by ichee** [comment]
VI Refresh Rate set to 1800 fixes intro sync, at least when using Azimer's audio plugin.

**Comment by thecoreyburton** [comment]
Is this a similar issue to the audio being out of sync in regards to the Super Smash Bros. intro?
